startupWizard.h

```diff
@@ -0,0 +1,31 @@
+//=============================================================================
+//  MuseScore
+//  Linux Music Score Editor
+//  $Id: capella.h 3833 2011-01-04 13:55:40Z wschweer $
+//
+//  Copyright (C) 2009-2013 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef startupWizard_h
+#define startupWizard_h
+
+class StartupWizard : public QInputDialog
+{
+      public:
+      StartupWizard();
+      ~StartupWizard();
+      static QString layoutInput();
```

Comment:
Hmm, it isn't added to CMakeListe.txt, so doesn't get compiled at all and then missing at link time. Sorry, didn't spot that earlier